frontend/gahadavala-dynasty/gahadavala.css

```
@@ -0,0 +1,1559 @@
+      circle at 90% 30%,
+      rgba(215, 168, 77, 0.08),
+      transparent 30%
+    ),
+    linear-gradient(
+      180deg,
+      #100b07,
+      #170e09 45%,
+      #0d0906
+    );
+
+  color: var(--gah-cream);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)